apps/build-onchain-apps/app/buy-me-coffee/page.tsx

```diff
@@ -0,0 +1,13 @@
+import { generateMetadata } from '../../src/utils/generateMetadata';
+import BuyMeCoffeePage from '.';
+
+export const metadata = generateMetadata({
```

Comment:
soooo, medatada just gets absorbed automatically? would you mind link a doc on this.

Reply:
Yep

https://nextjs.org/docs/app/building-your-application/optimizing/metadata

I'd say they made it worse since now it's pretty hard to get router info to include in the metadata, which is why we have to pass the pathname explicitly now